front/package-lock.json

```diff
@@ -1,16582 +0,0 @@
-{
```

Comment:
Looks like front/package-lock.json got deleted too in addition to the now-removed package-lock.json. I recommend resetting the latest commit with

git reset --soft HEAD~1